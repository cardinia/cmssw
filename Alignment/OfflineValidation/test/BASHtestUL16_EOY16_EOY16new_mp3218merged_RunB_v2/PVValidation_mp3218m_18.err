26-Sep-2019 12:38:36 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/00000/A026521D-0507-E911-ABB1-1866DA7F8E98.root
26-Sep-2019 12:38:41 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/00000/A026521D-0507-E911-ABB1-1866DA7F8E98.root
Begin processing the 1st record. Run 273410, Event 1320724, LumiSection 1 on stream 0 at 26-Sep-2019 12:38:56.755 CEST
Begin processing the 1001st record. Run 273410, Event 8321247, LumiSection 5 on stream 0 at 26-Sep-2019 12:44:15.196 CEST
Begin processing the 2001st record. Run 273410, Event 13881314, LumiSection 9 on stream 0 at 26-Sep-2019 12:48:20.653 CEST
Begin processing the 3001st record. Run 273410, Event 21112039, LumiSection 13 on stream 0 at 26-Sep-2019 12:52:05.236 CEST
Begin processing the 4001st record. Run 273410, Event 30622844, LumiSection 18 on stream 0 at 26-Sep-2019 12:56:10.773 CEST
Begin processing the 5001st record. Run 273410, Event 36431944, LumiSection 22 on stream 0 at 26-Sep-2019 13:00:00.881 CEST
Begin processing the 6001st record. Run 273410, Event 43792533, LumiSection 26 on stream 0 at 26-Sep-2019 13:04:19.810 CEST
Begin processing the 7001st record. Run 273410, Event 50410843, LumiSection 30 on stream 0 at 26-Sep-2019 13:08:38.315 CEST
Begin processing the 8001st record. Run 273410, Event 59681862, LumiSection 35 on stream 0 at 26-Sep-2019 13:13:05.121 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:13:59 CEST  Run: 273410 Event: 61571251
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 9001st record. Run 273410, Event 66901802, LumiSection 39 on stream 0 at 26-Sep-2019 13:17:26.649 CEST
Begin processing the 10001st record. Run 273410, Event 73653154, LumiSection 43 on stream 0 at 26-Sep-2019 13:21:46.585 CEST
Begin processing the 11001st record. Run 273410, Event 80470492, LumiSection 47 on stream 0 at 26-Sep-2019 13:26:03.791 CEST
Begin processing the 12001st record. Run 273410, Event 87984928, LumiSection 52 on stream 0 at 26-Sep-2019 13:30:20.927 CEST
Begin processing the 13001st record. Run 273410, Event 95280210, LumiSection 56 on stream 0 at 26-Sep-2019 13:34:32.607 CEST
Begin processing the 14001st record. Run 273410, Event 101785163, LumiSection 60 on stream 0 at 26-Sep-2019 13:38:27.532 CEST
Begin processing the 15001st record. Run 273410, Event 108398835, LumiSection 64 on stream 0 at 26-Sep-2019 13:42:08.963 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:44:39 CEST  Run: 273410 Event: 114341035
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 16001st record. Run 273410, Event 118291180, LumiSection 69 on stream 0 at 26-Sep-2019 13:45:54.007 CEST
Begin processing the 17001st record. Run 273410, Event 124615859, LumiSection 73 on stream 0 at 26-Sep-2019 13:49:16.311 CEST
Begin processing the 18001st record. Run 273410, Event 131854156, LumiSection 77 on stream 0 at 26-Sep-2019 13:52:35.579 CEST
Begin processing the 19001st record. Run 273410, Event 139404887, LumiSection 82 on stream 0 at 26-Sep-2019 13:55:54.057 CEST
Begin processing the 20001st record. Run 273410, Event 147479553, LumiSection 86 on stream 0 at 26-Sep-2019 13:59:38.497 CEST
Begin processing the 21001st record. Run 273449, Event 257741499, LumiSection 178 on stream 0 at 26-Sep-2019 14:02:45.524 CEST
Begin processing the 22001st record. Run 273449, Event 265663986, LumiSection 183 on stream 0 at 26-Sep-2019 14:02:48.443 CEST
Begin processing the 23001st record. Run 273449, Event 272617141, LumiSection 188 on stream 0 at 26-Sep-2019 14:02:51.764 CEST
Begin processing the 24001st record. Run 273449, Event 281800722, LumiSection 194 on stream 0 at 26-Sep-2019 14:02:54.510 CEST
Begin processing the 25001st record. Run 273449, Event 289008142, LumiSection 199 on stream 0 at 26-Sep-2019 14:02:57.515 CEST
Begin processing the 26001st record. Run 273449, Event 295603643, LumiSection 204 on stream 0 at 26-Sep-2019 14:03:00.120 CEST
Begin processing the 27001st record. Run 273449, Event 304051438, LumiSection 209 on stream 0 at 26-Sep-2019 14:03:03.074 CEST
Begin processing the 28001st record. Run 273725, Event 2413060787, LumiSection 1659 on stream 0 at 26-Sep-2019 14:03:05.439 CEST
Begin processing the 29001st record. Run 273725, Event 2441061139, LumiSection 1679 on stream 0 at 26-Sep-2019 14:03:07.740 CEST
Begin processing the 30001st record. Run 273725, Event 2450188789, LumiSection 1686 on stream 0 at 26-Sep-2019 14:03:10.028 CEST
Begin processing the 31001st record. Run 273725, Event 2458941941, LumiSection 1693 on stream 0 at 26-Sep-2019 14:03:12.379 CEST
Begin processing the 32001st record. Run 273725, Event 2467291141, LumiSection 1699 on stream 0 at 26-Sep-2019 14:03:14.812 CEST
Begin processing the 33001st record. Run 273725, Event 2477494297, LumiSection 1706 on stream 0 at 26-Sep-2019 14:03:17.223 CEST
Begin processing the 34001st record. Run 273725, Event 2486174017, LumiSection 1713 on stream 0 at 26-Sep-2019 14:03:20.065 CEST
Begin processing the 35001st record. Run 273725, Event 2494909463, LumiSection 1719 on stream 0 at 26-Sep-2019 14:03:22.760 CEST
Begin processing the 36001st record. Run 273725, Event 2504223383, LumiSection 1726 on stream 0 at 26-Sep-2019 14:03:24.865 CEST
Begin processing the 37001st record. Run 273725, Event 2512436645, LumiSection 1732 on stream 0 at 26-Sep-2019 14:03:27.297 CEST
Begin processing the 38001st record. Run 273725, Event 2524091085, LumiSection 1740 on stream 0 at 26-Sep-2019 14:03:29.943 CEST
26-Sep-2019 14:03:30 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/00000/A026521D-0507-E911-ABB1-1866DA7F8E98.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       2        2
    2 fileAction           -s file_close                             1        1
    3 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          273410/61571251  273410/114341035 
    2 fileAction           PostGlobalEndRun                  
    3 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 2                   2
System                  3                   3

dropped waiting message count 0
[8.991MB/8.991MB][100%][==================================================][8.991MB/s]  [8.991MB/8.991MB][100%][==================================================][8.991MB/s]  
