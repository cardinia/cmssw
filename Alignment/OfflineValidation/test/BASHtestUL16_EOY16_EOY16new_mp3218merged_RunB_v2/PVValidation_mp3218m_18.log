000 (3189526.018.000) 09/26 11:22:16 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189526.018.000) 09/26 12:37:46 Started transferring input files
	Transferring to host: <128.142.113.125:9618?addrs=128.142.113.125-9618+[2001-1458-301-90--100-9b]-9618&noUDP&sock=14533_9028_12024>
...
040 (3189526.018.000) 09/26 12:37:46 Finished transferring input files
...
001 (3189526.018.000) 09/26 12:37:46 Job executing on host: <128.142.113.125:9618?addrs=128.142.113.125-9618+[2001-1458-301-90--100-9b]-9618&noUDP&sock=14396_097d_3>
...
006 (3189526.018.000) 09/26 12:37:55 Image size of job updated: 19892
	14  -  MemoryUsage of job (MB)
	13396  -  ResidentSetSize of job (KB)
...
006 (3189526.018.000) 09/26 12:42:56 Image size of job updated: 593808
	539  -  MemoryUsage of job (MB)
	551648  -  ResidentSetSize of job (KB)
...
006 (3189526.018.000) 09/26 12:47:56 Image size of job updated: 596340
	542  -  MemoryUsage of job (MB)
	554180  -  ResidentSetSize of job (KB)
...
006 (3189526.018.000) 09/26 12:52:57 Image size of job updated: 596340
	542  -  MemoryUsage of job (MB)
	554804  -  ResidentSetSize of job (KB)
...
006 (3189526.018.000) 09/26 12:57:58 Image size of job updated: 596340
	543  -  MemoryUsage of job (MB)
	555064  -  ResidentSetSize of job (KB)
...
006 (3189526.018.000) 09/26 13:07:58 Image size of job updated: 596340
	543  -  MemoryUsage of job (MB)
	555080  -  ResidentSetSize of job (KB)
...
006 (3189526.018.000) 09/26 13:18:00 Image size of job updated: 596340
	543  -  MemoryUsage of job (MB)
	555288  -  ResidentSetSize of job (KB)
...
006 (3189526.018.000) 09/26 13:28:01 Image size of job updated: 596340
	544  -  MemoryUsage of job (MB)
	556564  -  ResidentSetSize of job (KB)
...
006 (3189526.018.000) 09/26 13:38:02 Image size of job updated: 596340
	545  -  MemoryUsage of job (MB)
	557208  -  ResidentSetSize of job (KB)
...
006 (3189526.018.000) 09/26 13:43:02 Image size of job updated: 596340
	546  -  MemoryUsage of job (MB)
	558744  -  ResidentSetSize of job (KB)
...
040 (3189526.018.000) 09/26 14:03:56 Started transferring output files
...
040 (3189526.018.000) 09/26 14:03:56 Finished transferring output files
...
005 (3189526.018.000) 09/26 14:03:56 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 01:22:03, Sys 0 00:00:10  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:22:03, Sys 0 00:00:10  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	8209  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	8209  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     9252        1   2118049 
	   Memory (MB)          :      546     2000      2000 
...
