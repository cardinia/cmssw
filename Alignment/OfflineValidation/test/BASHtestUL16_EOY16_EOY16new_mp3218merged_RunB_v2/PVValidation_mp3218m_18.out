Job started at  Thu Sep 26 12:37:47 CEST 2019
batch dir: /pool/condor/dir_28851 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_273410_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 9.1M -rw-r--r--. 1 tkello zh 9.0M Sep 26 14:03 PVValidation_mp3218m_273410.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:37 PVValidation_mp3218m_273410_cfg.py -rw-r--r--. 1 tkello zh 6.8K Sep 26 14:03 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:38 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 26 12:37 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:38 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:37 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:37 var
Job ended at  Thu Sep 26 14:03:56 CEST 2019
